priam/src/main/java/com/netflix/priam/backup/BackupRestoreUtil.java

```diff
@@ -58,14 +59,13 @@ public static Optional<AbstractBackupPath> getLatestValidMetaPath(
                 .findFirst();
     }
 
-    public static List<AbstractBackupPath> getAllFiles(
+    public static List<AbstractBackupPath> getMostRecentSnapshotPaths(
```

Comment:
renaming the method was a great idea, makes it more precise (now that it's different) and also makes it impossible to forget to update a callsite